src/components/blocks/ovc/dvm-application-selection.js

```diff
@@ -0,0 +1,70 @@
+import React from "react"
+import { StaticQuery, graphql } from "gatsby"
+import { Container, Row } from "react-bootstrap"
+import {getImage} from "gatsby-plugin-image"
+import Overlay from "components/shared/overlay"
+import styled from 'styled-components'
+
+const yaml = require('js-yaml');
+
+const colourOptions = {background: "var(--uog-yellow)", colour: "#000000"};
+
+const render = ({ field_yaml_map, relationships }, colourOptions) => {
+  let yamlMap;
+  let yamlFiles = {};
+  relationships.field_yaml_files.forEach(file => {
+    yamlFiles[file.path.alias] = file.relationships.field_media_image.localFile;
+  });
+  
+  try {
+    yamlMap = yaml.load(field_yaml_map);
+  } catch (e) {
+    console.log(e);
+    return null;
+  }
+  
+  return (
+    <>
+        <div className="d-flex flex-column bg-light">
+          <Overlay.GatsbyImage gatsbyImageData={getImage(yamlFiles[yamlMap.image.src])} alt={yamlMap.image.alt}>
+            <Container className="page-container">
+              <Row className="h-100 w-100 p-5 justify-content-center align-items-center">
+                  <p className="display-2 text-dark"> <i class="fa-light fa-circle-3 fa-lg"> </i> <strong>{yamlMap.title}</strong></p>
+              </Row>
+            </Container>
+          </Overlay.GatsbyImage>
+        </div>
+    </>
+  )}
+
+const query = graphql`
+  query {
+    blockContentYamlBlock(field_yaml_id: {glob: "dvm_application_selection"}) {
+      id
+      field_yaml_id
+      field_yaml_map
+      relationships {
+        field_yaml_files {
+          id
+          name
+          relationships {
+            field_media_image {
+              localFile {
+                childImageSharp {
+                  gatsbyImageData(width: 1400, height: 190, placeholder: BLURRED, layout: CONSTRAINED)
+                }
+              }
+            }
+          }
+          path {
+            alias
+          }
+        }
+      }
+    }
+  }
+`
+
+export default function DVMApplicationSelection() {
```

Comment:
```suggestion
export default function DVMApplication (props) {
```
Example if we changed this file to dvm-application.js and passed props to component